.github/workflows/wheel_tests_and_release.yml

```diff
@@ -0,0 +1,193 @@
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel
+
+      - name: Download FFTW libraries
+        run: |
+          Write-Output "Downloading FFTW release (64 bit)"
+          Invoke-WebRequest -Uri "https://fftw.org/pub/fftw/fftw-3.3.5-dll64.zip" -OutFile "${{ github.workspace }}\fftw.zip"
+          Write-Output "Unzipping FFTW release"
+          Expand-Archive -Path "${{ github.workspace }}\fftw.zip" -DestinationPath "${{ github.workspace }}\pyfftw"
+        shell: powershell
+
+        # Annoyingly these two next steps are needed as FFTW does not ship with
+        # .lib files which msvc requires for linking
+      - uses: ilammy/msvc-dev-cmd@v1
+        name: Setup Visual Code build tools
+
+      - name: Convert .def files to .lib
+        run: |
+          lib.exe /def:${{ github.workspace }}\pyfftw\libfftw3-3.def /out:${{ github.workspace }}\pyfftw\libfftw3-3.lib /machine:x64
+          lib.exe /def:${{ github.workspace }}\pyfftw\libfftw3f-3.def /out:${{ github.workspace }}\pyfftw\libfftw3f-3.lib /machine:x64
+          lib.exe /def:${{ github.workspace }}\pyfftw\libfftw3l-3.def /out:${{ github.workspace }}\pyfftw\libfftw3l-3.lib /machine:x64
+
+      - name: Build Windows wheels for CPython (Win64)
+        run: |
+          python -m cibuildwheel --output-dir dist
+
+      - uses: actions/upload-artifact@v2
+        name: Upload wheels as artifacts
+        with:
+          name: wheels
+          path: ./dist/*.whl
+
+
+  deploy:
+    name: Release
+    needs: [build_linux_wheels, build_macos_wheels, build_windows_wheels]
+    if: github.repository_owner == 'pyFFTW' && startsWith(github.ref, 'refs/tags/v') && always()
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Install Twine
+        run: |
+          python -m pip install --upgrade pip
+          pip install twine
+
+      - uses: actions/download-artifact@v2
+        id: download
+        with:
+          name: wheels
+          path: ./dist
+
+      - name: Publish the source distribution on PyPI
+        run: |
+          PYFFTW_VERSION=$(git describe --tags)
+          python setup.py sdist
+          ls -la ${{ github.workspace }}/dist
+          # We prefer to release wheels before source because otherwise there is a
+          # small window during which users who pip install pyfftw will require compilation.
+          twine upload ${{ github.workspace }}/dist/*.whl
+          twine upload ${{ github.workspace }}/dist/pyfftw-${PYFFTW_VERSION:1}.tar.gz
+        env:
+          TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
```

Comment:
@hgomersall, can you add `TWINE_USERNAME` and `TWINE_PASSWORD` as secrets on this repository? The process to add them is going to the "Settings" for the repository and selecting "Secrets" from the menu at the left. I do not have the permissions required to do it.

If they are not defined, automated deployment will fail, but we can still manually download the artifacts and upload the release to PyPI via `twine`.

Reply:
Is this the PyPI API tokens?

Reply:
It looks like there is a TestPyPI as well as a PyPI. I think the name should be different here (not twine, but TestPyPI, if that's what is needed).